components/miller/index.css

```diff
@@ -11,28 +11,31 @@ governing permissions and limitations under the License.
 */
 
 .spectrum-MillerColumns {
-  --spectrum-millercolumn-margin: 8px;
+  --spectrum-millercolumn-inline-size: 272px;
+  --spectrum-millercolumn-padding: var(--spectrum-spacing-100);
+  --spectrum-millercolumn-margin-left: var(--spectrum-spacing-100);
+  --spectrum-millercolumn-margin-right: var(--spectrum-spacing-100);
```

Comment:
These two could be renamed with logical property naming? e.g. `--spectrum-millercolumn-margin-inline-start`